hetnet/permuted/matrix/README.md

```diff
@@ -0,0 +1,5 @@
+## Files
+
+[`hetionet-v1.0-permutations.zip`](hetionet-v1.0-permutations.zip) contains 200 permuted hetnets derived from Hetionet v1.0 using the XSwap algorithm.
+Permutations 001 through 005 were generated by [Project Rephetio](https://doi.org/10.7554/eLife.26726 "Systematic integration of biomedical knowledge prioritizes drugs for repurposing. eLife 2017")
+Permutations 006 through 200 were generated as part of the [`greenelab/hetmech`](https://github.com/greenelab/hetmech/pull/127) project.
```

Comment:
@zietzm the [`data/hetionet-v1.0.hetmat/02.generate-permutations.ipynb`](https://github.com/greenelab/hetmech/blob/34e95b9f72f47cdeba3d51622bee31f79e9a4cb8/data/hetionet-v1.0.hetmat/02.generate-permutations.ipynb) only includes info for 25 permutations. Do you have the notebook / stats.tsv for all 200 permutations? Is this online somewhere?